lms/djangoapps/courseware/features/lti.feature

```diff
@@ -48,7 +48,7 @@ Feature: LMS.LTI component
   | False              | 10     | True      | True      |
   And I submit answer to LTI 1 question
   And I click on the "Progress" tab
-  Then I see text "Problem Scores: 5/10"
+  Then I see text "5/10"
```

Comment:
Do you understand why "Problem Scores: " had to be dropped? I don't see that the HTML structure changed (though unfortunately the whole thing isn't good from an i18n perspective).